[
 {
  "author": "dlr", 
  "date": "2006-04-13T20:07:54.739265Z", 
  "msg": [
   "Create backport branch for r19348."
  ], 
  "revision_id": "859425"
 }, 
 {
  "author": "dlr", 
  "date": "2006-04-13T20:06:25.472735Z", 
  "msg": [
   "* STATUS: Vote for r19348."
  ], 
  "revision_id": "859424"
 }, 
 {
  "author": "rooneg", 
  "date": "2006-04-13T19:28:02.986465Z", 
  "msg": [
   "* STATUS: Add r19348."
  ], 
  "revision_id": "859423"
 }, 
 {
  "author": "rooneg", 
  "date": "2006-04-13T19:25:02.044565Z", 
  "msg": [
   "Fix a crash that occurs when running 'svn status -u' with a missing", 
   "subdirectory.", 
   "Found by: Fabian Cenedese <Cenedese@indel.ch>", 
   "* subversion/libsvn_wc/status.c", 
   "(find_dir_url): Handle the case where an entry's url is NULL, and", 
   "add a comment describing how this can happen.", 
   "* subversion/tests/cmdline/stat_tests.py", 
   "Import shutil.", 
   "(status_dash_u_missing_dir): New test for this problem.", 
   "(test_list): Add new test."
  ], 
  "revision_id": "859422"
 }, 
 {
  "author": "jerenkrantz", 
  "date": "2006-04-13T18:48:28.710396Z", 
  "msg": [
   "ra_serf: When we see a 5xx that also closes the socket (the default behavior),", 
   "ensure that we reset our connection.", 
   "Combined with serf r1036, fixes commit test #27 (commit with conflicts and", 
   "check txn in repo).  (We weren't deleting the activity after the error.)", 
   "* subversion/libsvn_ra_serf/util.c", 
   "(handler_default): If we see a 5xx and a Conn: Close, reset our connection", 
   "so it can be reused later."
  ], 
  "revision_id": "859421"
 }
]